api.planx.uk/modules/webhooks/sendNotification/service.ts

```diff
@@ -0,0 +1,59 @@
+import { Passport } from "@opensystemslab/planx-core";
+import SlackNotify from "slack-notify";
+import {
+  BOPSEventData,
+  EmailEventData,
+  EventData,
+  EventType,
+  UniformEventData,
+} from "./types";
+import { $admin } from "../../../client";
+
+export const sendSlackNotification = async (
+  data: EventData,
+  type: EventType,
+) => {
+  const slack = SlackNotify(process.env.SLACK_WEBHOOK_URL!);
+  let message = getMessageForEventType(data, type);
+
+  const sessionId = getSessionIdFromEvent(data, type);
+  const feePayable = await getFeePayableForSession(sessionId);
+  if (!feePayable) message += " [Exempt]";
```

Comment:
This PR ended up being more refactor than feature sorry, but this is the core of the new work.

Reply:
I'm a little hesitant about this `(!feePayable)` logic because our most recent bug around this was actually content incorrectly setting fee payable to 0 regardless of exempt/resubmission answers. I _think_ this change would have actually incorrectly labeled those cases `[Exempt]` and we perhaps wouldn't have caught that as quick? 

My hunch is something like this will be more reliable / robust:
- Regardless of `application.fee.payable` & `application.fee.calculated`...
- If `application.fee.exemption.disability` is `true` then `message += " [Exempt]"`
- If `application.fee.exemption.resubmission` is `true` then `message += " [Resubmission]"`
- ** Exemptions are asked about via a Question node so you'll want to do `Boolean({field}[0])` to evaluate

So, the only cases going forward where we'd manually look into a case are: 
- Missing payment notification && submission notification does _not_ have `[Exempt]` or `[Resubmission]` suffix

Does this make sense?

Reply:
This is a really helpful comment thanks 👍 I'll go ahead and make this change.

Reply:
PR updated with these changes @jessicamcinchak 👌